include/NotificationsManager.php

```diff
@@ -0,0 +1,135 @@
+<?php
+/**
+ * @file include/NotificationsManager.php
+ */
+require_once("include/datetime.php");
+require_once("include/bbcode.php");
+
+/**
+ * @brief Read and write notifications from/to database
+ */
+class NotificationsManager {
+    private $a;
+    
+    public function __construct() {
+        $this->a = get_app();
+    }
+    
+	/**
+	 * @brief set some extra note properties
+	 *
+	 * @param array $notes array of note arrays from db
+	 * @return array Copy of input array with added properties
+	 * 
+	 * Set some extra properties to note array from db:
+	 *  - timestamp as int in default TZ
+	 *  - date_rel : relative date string
+	 *  - msg_html: message as html string
+	 *  - msg_plain: message as plain text string
+	 */
+    private function _set_extra($notes) {
+        $rets = array();
+        foreach($notes as $n) {
+            $local_time = datetime_convert('UTC',date_default_timezone_get(),$n['date']);
+            $n['timestamp'] = strtotime($local_time);
+            $n['date_rel'] = relative_date($n['date']);
+			$n['msg_html'] = bbcode($n['msg'], false, false, false, false);
+			$n['msg_plain'] = explode("\n",trim(html2plain($n['msg_html'], 0)))[0];
```

Comment:
the html2plain function is unknown at this point on my node which results in a fatal error when following a notification to a posting.